lib/ecto/pagination.ex

```diff
@@ -0,0 +1,79 @@
+defmodule PhoenixApiToolkit.Ecto.Pagination do
+  @moduledoc """
+  Helper functions for pagination.
+  """
+  import Ecto.Query
+
+  @doc """
+  Paginate the query by specifying a limit and/or offset.
```

Comment:
Hier moet duidelijk bij komen te staan dat het niet gebruikt kan worden icm composite primary keys, en dat het niet nodig is om dit te gebruiken indien er geen joins in de base query zitten. Ik denk dat we zelfs de functienaam moeten veranderen in paginate_binding, zie de overige comments